src/BlockingDriver.php

```diff
@@ -348,7 +348,7 @@ public function chown(string $path, int $uid, int $gid): Promise
             return new Failure(new FilesystemException($message));
         }
 
-        return new Success;
+        return new Success(true);
```

Comment:
true is returned in other functions like symlink, link and return. Also it seems that other drivers can return a boolean for chown so I added this for consistency.

Reply:
Not sure whether we should change that in a minor. What do other drivers currently return? Could you add a test for this, so all drivers behave equally?

Reply:
EioDriver [returns true](https://github.com/amphp/file/blob/9e8bb4defe791f41e56b98193a4399d7083c5ee8/src/EioDriver.php#L347) and UvDriver [returns boolean](https://github.com/amphp/file/blob/master/src/UvDriver.php#L556). Not sure if UvDriver can actually return false or when it would happen.

> Could you add a test for this, so all drivers behave equally?

I'm not exactly sure how. To test chown you first need some user to chown to... also it needs root privileges, doesn't it? I'm guessing that's why there is no test for chown at the moment.

Reply:
I guess we could chown from the current user to the current user?

Reply:
Not sure but I guess I can try to add such test to see how that behaves.

Reply:
@kelunik Sorry, I kinda forgot about this PR. Added the test just now.